e2e-tests/native/capabilities.ts

```diff
@@ -41,7 +42,7 @@ export default {
       'appium:deviceName': 'iPhone 8',
       'appium:automationName': 'XCUITest',
       // @ts-ignore unknown property
-      'appium:waitForQuiescence': true
+      'appium:waitForIdleTimeout': 10000
```

Comment:
`waitForIdleTimeout` is very similar to `waitForQuiescence` as it waits for the timeout or idle by itself. I replaced those two as webdriver seems to translate unknown booleans to strings -.-